analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt

```diff
@@ -140,6 +140,36 @@ class MavenSupportTest : WordSpec({
                 path = "boringssl-static"
             )
         }
+
+        "handle GitHub URLs with double 'git:' prefix" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    connection = "scm:git:git:github.com/MarkusAmshove/Kluent.git"
```

Comment:
Could it be that here, the `//` has just been forgotten, so it should be handled like `scm:git:git://` in 132 which maps it to the `git` protocol?

Reply:
Of course I don't know for sure, but even if that was the case, we're better off with HTTP(S) because, as you known, [GitHub deprecated the `git://` protocol](https://github.blog/2021-09-01-improving-git-protocol-security-github/#no-more-unauthenticated-git).

Reply:
Yeah, I know. Was just wondering whether to keep on mapping that at a single place. Anyhow, let's keep this as is. It's probably better.